doc/snippets/Microsoft.Data.SqlClient/SqlCommandBuilder.xml

```diff
@@ -1,439 +1,530 @@
+      </summary>
+      <value>
+        A <see cref="T:Microsoft.Data.SqlClient.SqlDataAdapter" /> object.
+      </value>
+      <remarks>
+        <para>
+          The <see cref="T:Microsoft.Data.SqlClient.SqlCommandBuilder" /> registers itself as a listener for <see cref="E:Microsoft.Data.SqlClient.SqlDataAdapter.RowUpdating" /> events that are generated by the <see cref="T:Microsoft.Data.SqlClient.SqlDataAdapter" /> specified in this property.
+        </para>
+        <para>
+          When you create a new instance of <see cref="T:Microsoft.Data.SqlClient.SqlCommandBuilder" />, any existing <see cref="T:Microsoft.Data.SqlClient.SqlCommandBuilder" /> associated with this <see cref="T:Microsoft.Data.SqlClient.SqlDataAdapter" /> is released.
+        </para>
+      </remarks>
+    </DataAdapter>
+    <DeriveParameters>
+      <param name="command">
+        The <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> referencing the stored procedure from which the parameter information is to be derived. The derived parameters are added to the <see cref="P:Microsoft.Data.SqlClient.SqlCommand.Parameters" /> collection of the <see cref="T:Microsoft.Data.SqlClient.SqlCommand" />.
+      </param>
+      <summary>
+        Retrieves parameter information from the stored procedure specified in the <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> and populates the <see cref="P:Microsoft.Data.SqlClient.SqlCommand.Parameters" /> collection of the specified <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> object.
+      </summary>
+      <remarks>
+        <para>
+          <c>DeriveParameters</c> overwrites any existing parameter information for the <see cref="T:Microsoft.Data.SqlClient.SqlCommand" />.
+        </para>
+        <para>
+          <c>DeriveParameters</c> requires an additional call to the database to obtain the information. If the parameter information is known in advance, it is more efficient to populate the parameters collection by setting the information explicitly.
+        </para>
+        <para>
+          You can only use <c>DeriveParameters</c> with stored procedures. You cannot use <c>DeriveParameters</c> with extended stored procedures. You cannot use <c>DeriveParameters</c> to populate the <see cref="T:Microsoft.Data.SqlClient.SqlParameterCollection" /> with arbitrary Transact-SQL statements, such as a parameterized SELECT statement.
+        </para>
+        <para>
+          For more information, see <see href="/sql/connect/ado-net/configure-parameters">Configuring parameters</see>.
+        </para>
+      </remarks>
+      <exception cref="T:System.InvalidOperationException">
+        The command text is not a valid stored procedure name.
+      </exception>
+    </DeriveParameters>
+    <Dispose>
+      <param name="disposing">
+        To be added.
+      </param>
+      <summary>
+        To be added.
+      </summary>
+      <remarks>
+        To be added.
```

Comment:
Were you going to rewrite these remarks based on available formatted CDATA?

Reply:
I would, but there isn't any in main...
![image](https://github.com/user-attachments/assets/d68f7ead-e209-4dee-b04e-f0fed608410b)

I've stripped out <remarks>To be added</remarks> for ones that have other comments (basically under the validation that if we haven't needed remarks up to now, we probably won't ever need them). But if there aren't any comments other than to be added, then I opted to keep it 1) so we have some comments and 2) so we can maybe one day go through and add some. But, writing them now is outside the scope of these PRs.

Reply:
Sure, thanks!